apps/frontend/src/components/Auth/apiClient.tsx

```diff
@@ -86,17 +93,55 @@ export class ApiClient {
     return this.get("/auth/timesheet") as Promise<string>;
   }
 
+
+  // a function to create user object from a passed in userId
+  public async createUser(userId: String): Promise<UserSchema> {
```

Comment:
I think `createUser` might be misleading as a function name - it seems like this is the `getUser` function from down below. Can you make this the `getUser` function, delete the wrapper down below, and then call this function in `getUsers`?